src/NCalc.Async/NCalc.Async.csproj

```diff
@@ -13,6 +13,10 @@
         <Description>NCalcAsync is a fully async port of the NCalc.NCalc is a fast and lightweight expression evaluator library for .NET, designed for flexibility and high performance. It supports a wide range of mathematical and logical operations. NCalc can parse any expression and evaluate the result, including static or dynamic parameters and custom functions.</Description>
     </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0' OR '$(TargetFramework)' == 'net10.0'">
```

Comment:
I don't like this condition, because we need to add new framework after each release. 

It should be better to use this one:

```
<PropertyGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
  <PublishTrimmed>true</PublishTrimmed>
</PropertyGroup>
```
Btw, should we use `IsAotCompatible` instead of `PublishTrimmed`?